langchain/src/chat_models/llama_cpp.ts

```diff
@@ -29,8 +30,14 @@ export interface LlamaCppInputs
 export interface LlamaCppCallOptions extends BaseLanguageModelCallOptions {
   /** The maximum number of tokens the response should contain. */
   maxTokens?: number;
-  /** A function called when matching the provided token array */
-  onToken?: (tokens: number[]) => void;
+  /** This lets the streaming method know which model we are using,
+   *  valid options are:
+   *  'llama2'	- A Llama2 model, this is the default.
+   *  'chatML'  - A ChatML model
+   *  'falcon'	- A Falcon model
+   *  'genral'  - Any other model, uses "### Human\n", "### Assistant\n" format
+   */
+  streamingModel?: string;
```

Comment:
Use a union type or a check in the constructor that verifies the input is one of these?